docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Add Google Service Account Keys using FCM V1'
+sidebar_title: Add Android FCM V1 credentials
+description: Learn how to create a Google Service Account Key for sending Android Notifications using FCM V1
+---
+
+import { Step } from '~/ui/components/Step';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Tab, Tabs } from '~/ui/components/Tabs';
+
+## Create a new Google Service Account Key
+
+Here are the steps to configure a new Google Service Account Key in EAS for sending Android Notifications using FCM V1.
+
+<Step label="1">
+  Create a new Firebase project for your app in the [Firebase Console](https://console.firebase.google.com). If you already have a Firebase project for your app, continue to the next step.
+
+  <ImageSpotlight
+    alt="Create a project in the Firebase Console"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/01-new-firebase-project.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="2">
+  In the Firebase console, open Settings > [Service Accounts](https://console.firebase.google.com/project/_/settings/serviceaccounts/adminsdk) for your project.
+
+  <ImageSpotlight
+    alt="Firebase Console > Settings > Service Accounts"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/02-manage-service-accounts.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="3">
+  Click **Generate New Private Key**, then confirm by clicking **Generate Key**. Securely store the JSON file containing the private key.
+
+  <ImageSpotlight
+    alt="Generate a new private key in the Firebase Console"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/03-generate-key.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="4">
+  Upload the JSON file to EAS and configure it for sending Android notifications. This can be done in EAS CLI or via [expo.dev](https://expo.dev).
```

Comment:
```suggestion
  Upload the JSON file to EAS and configure it for sending Android notifications. This can be done using EAS CLI or in [Expo dashboard](https://expo.dev).
```

We usually call the main interface of expo.dev, Expo dashboard. For consistency, let's use that here too.